javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,97 @@
+//    Formal fullname
+
+
+ function getFullname( firstname, surname, useFormalName) {
+    if ( firstname === "" && surname ===""){
+        return "Please fill the form fully";
+    }
+    if ( useFormalName ) {
+        return '"Lord '+firstname+" "+surname+'"' ;
+    } else {
+   return '"'+firstname+" "+surname+'"' ;
+}}
+const fullname1 = getFullname("Benjamin","Hughes",true);
+const fullname2 = getFullname("Ali","Ehsan");
+console.log(fullname1);
+console.log(fullname2); 
+
+
+//     Event application
+ function getEventWeekday( eventDay ) {
+    const today = new Date();
+    const date = today.getDay();
+    const weekday =["Sunday", "Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"] ;
+    let dayOfWeek  = weekday[date];
+    let dayOfEvent;
+    eventDay += date;
+
+if( eventDay >= 7  && eventDay%6 != 0){
+        eventDay = Math.abs(( eventDay % 6 )- (~~(eventDay/6)));
+     }
+
+else if ( eventDay%6 === 0  && eventDay >= 7 ){
+    eventDay = eventDay/6 +1;
+}
+dayOfEvent = weekday[eventDay];
+
+console.log("Today is " +'"'+dayOfWeek+'"' +" So event would be held on " + '"'+dayOfEvent+'"!' );
+}
+console.log(getEventWeekday(6)); 
```

Comment:
This gives undefined, as the above function does not return anything.instead of console.log("Today.......") use a return keyword.